qftasm.js

```diff
@@ -176,15 +176,15 @@ function step_code() {
         vals.push(val);
     }
     
+    PC[1] = PC[1] + 1;
     PC[0] = PC[1];
-    PC[1] = PC[0]+1;
-    if(RAMdisplay.indexOf(0) >= 0){ update_display(0); }

```

Comment:
Err, does this only update the RAM display for address 0 when it is explicitly written to?